tools/python/run_CIs_for_external_pr.py

```diff
@@ -93,6 +93,8 @@ def main():
         # checks
         "onnxruntime-python-checks-ci-pipeline",
         "onnxruntime-binary-size-checks-ci-pipeline",
+        # big models
+        "Big Models",
```

Comment:
```suggestion
        # other
        "Big Models",
        "Python format",
```

Seems to be another required CI

Reply:
"Python format" will run automatically when you click "Approve and run" button following "5 workflows awaiting approval" message in an external PR.

Reply:
Ah ok. thanks for the info.

Reply:
That 'Approve and run' doesn't seem to be consistent though. Not showing up on #19079 which is why I thought 'Python format' might need to be added here.

Reply:
"Python format" is not in Azure Pipelines. It is github Action so I think adding it here would not take effect.